src/t/d4.clj

```diff
@@ -0,0 +1,164 @@
+(ns t.d4
+  (:gen-class)
+  (:require
+    [clojure.string :as cljstr]
+    [clojure.set :as cljset]))
+
+(def sample
+"MMMSXXMASM
+MSAMXMSMSA
+AMXSXMAAMM
+MSAMASMSMX
+XMASAMXAMM
+XXAMMXXAMA
+SMSMSASXSS
+SAXAMASAAA
+MAMMMXMMMM
+MXMXAXMASX")
+
+(defn parse-input
+  [input]
+  (->> input
+       cljstr/split-lines
+       (map-indexed (fn [i l]
+                      (->> l
+                      (map-indexed (fn [ii letter] {letter [[i ii]]}))
+                      (apply merge-with into))))
+       (apply merge-with into)))
+
+
+(comment
```

Comment:
Any reason to keep it around?

Reply:
I worked hard for it, I want you to know